SeaPkg/Tools/GenSeaArtifacts/test_aux/README.md

```diff
@@ -22,21 +22,28 @@ simple interface with the following two arguments:
 As mentioned above, the `-c`, `--config` argument is to pass in a configuration file that contains the information from
 a run log of the given mm supervisor core.
 
-### LoadAddress
+### MSEG Base / MSEG Size
 
-LoadAddress is the first configuration that is needed, which is a string hex representation of the address the mm
-supervisor core was loaded to during the build.
+A string hex representation of the base address and size of the MSEG reason during the run.
 
-`LoadAddress = "0x&BDC0000"`
+`MsegBase = "0x7BE00000"`
 
-### MmSupervisorCore
+`MsegSize = "0x1B6000"`
+
+### MmSupervisorBase
+
+A string hex representation of the address the mm supervisor core was loaded to during the run.
+
+`MmSupervisorBase = "0x7BDC0000"`
+
+### MmSupervisor
 
 MmSupervisorCore is a string dump of the entire post-execution MmSupervisorCore image. This is found in the build log

```

Comment:
For consistencies sake I think this should be MmSupervisor.

Reply:
I agree. Removed "core" so it is just "MmSupervisor" in a few places. Thanks :)